### PR TITLE
Remove libav dependency

### PR DESCRIPTION
**Pull request by at-wat**
To remove unnecessary dependency to video codecs.

### CODE REVIEW

**Comment by at-wat** [comment]
@f-fl0 please take a look

**Review by f-fl0 — COMMENTED** [review]
I think we should also change the default value of the `pixel_format` node parameter in https://github.com/seqsense/usb_cam/blob/fd765105c03be5545501e96afa675ba65a015121/nodes/usb_cam_node.cpp#L144.

**Comment by at-wat** [comment]
Changed default pixel_format to yuyv.

File: nodes/usb_cam_node.cpp [hunk 1]
```diff
@@ -141,7 +141,7 @@ class UsbCamNode
     node_.param("image_height", image_height_, 480);
     node_.param("framerate", framerate_, 30);
     // possible values: yuyv, uyvy, mjpeg, yuvmono10, rgb24
```
Comment (f-fl0) [inline nodes/usb_cam_node.cpp]: I guess we could also remove `mjpeg` from this comment.

**Review by f-fl0 — APPROVED** [review]
LGTM. Just left a comment.

File: nodes/usb_cam_node.cpp (hunk 1 above)
Comment (at-wat) [reply nodes/usb_cam_node.cpp]:
```suggestion
    // possible values: yuyv, uyvy, yuvmono10, rgb24
```

**Pull request merged**